src/grid/grid-filtering.spec.ts

```diff
@@ -778,10 +778,10 @@ describe("IgxGrid - Filtering actions", () => {
         filterIcon.nativeElement.click();
```

Comment:
Is there a test that tests changing filtering condition without an input?

Reply:
Nope, I am about to add tests.

Reply:
tests are added: https://github.com/IgniteUI/igniteui-angular/pull/1163/commits/ee75b51f50804f9ff78b73df64f9178a31b05318